src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,71 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
```

Comment:
Maybe you could add index.ts with exports of all types in one file. But it's not necessary